README.md

```diff
@@ -0,0 +1,52 @@
+
+![XMPPFramework](xmppframework.png)
+
+## XMPPFramework
+[![Build Status](https://travis-ci.org/robbiehanson/XMPPFramework.svg?branch=master)](https://travis-ci.org/robbiehanson/XMPPFramework) [![codecov.io](https://img.shields.io/badge/pod-v3.6.7-brightgreen.svg)](https://github.com/robbiehanson/XMPPFramework)
+
+An XMPP Framework in Objective-C for the Mac and iOS development community.
+
+### Abstract
+XMPPFramework provides a core implementation of RFC-3920 (the XMPP standard), along with the tools needed to read & write XML. It comes with multiple popular extensions (XEP's), all built atop a modular architecture, allowing you to plug-in any code needed for the job. Additionally the framework is massively parallel and thread-safe. Structured using GCD, this framework performs well regardless of whether it's being run on an old iPhone, or on a 12-core Mac Pro. (And it won't block the main thread... at all)
+
+### Install
+
+The easiest way to install the `XMPPFramework` is using CocoaPods:
+
+- This will install the whole framework with all the avaiable extensions:
+
+```
+pod 'XMPPFramework'
+
+```
+
+- If you want to pick what extensions to install:
+
+```
+pod 'XMPPFramework/XEP-0009'
+pod 'XMPPFramework/XEP-0224'
+```
+
+After `pod install` open the `.xcworkspace` and import:
+
+```
+import XMPPFramework      // swift
+#import <XMPPFramework.h> //objective-c 
```

Comment:
@andresinaka  Perhaps this should be `@import XMPPFramework;`? We might also want to mention that the minimum deployment target is iOS 8.0 / macOS 10.?? and it only works with `use_frameworks!`.